mycroft/skills/phillips_hue/vocab/en-us/ActivateSceneKeyword.voc

```diff
@@ -0,0 +1,6 @@
+activate scene
+activate seen
+set my lights to
+set scene
+set seen
```

Comment:
I added the alternate spelling after having the commands fail, and seeing that the cause was that 'seen' was used instead of 'scene.' I tested again just now to confirm. Here are the logs:
https://gist.github.com/ChristopherRogers1991/9d9f81958994dbd6212d2ad8d18183a0.

You'll notice that he correctly hears 'scene' the first few times, but when he hears 'seen,' it fails.